code/modules/research/protolathe.dm

```diff
@@ -145,9 +145,9 @@ Note: Must be placed west/left of and R&D console to function.
 	busy = 1
 	use_power(max(1000, (3750*amount/10)))
 	var/stacktype = stack.type
-	stack.use(amount)
-	if(do_after(user, 15, TRUE, 5, BUSY_ICON_FRIENDLY))
```

Comment:
Is the removal of the icon intended here?

Reply:
No visible message and the delay is pretty stupid I say.